File: 백준/Bronze/10798. 세로읽기/세로읽기.swift
// [Array<Array<String>>] matrix (mutable): 주어진 글자들을 저장하는 2차원 배열 (5행 고정)
var matrix: [[String]] = [[String]]()

// [Int] maxLength (mutable): matrix의 최대 행 길이
var maxLength: Int = 0

for _ in 0..<5 {
    // 입력된 문자열을 matrix에 저장
    let row: Array<String> = readLine()?.map { String($0) } ?? []
    matrix.append(row)
    maxLength = max(maxLength, row.count)
}

// 세로로 출력
for col in 0..<maxLength {
    for row in 0..<5 {
        // 만약 현재 인덱스가 존재한다면 출력
        if (matrix[row].count > col) {
            print(matrix[row][col], terminator: "")
        }
    }
}

File: 백준/Bronze/10798. 세로읽기/세로읽기.py
# [list(list(int))] matrix: 주어진 글자들을 저장하는 2차원 list
matrix = []

# 세로 길이는 5 고정
for row in range(5) :
    # [list] columnList: 영어 대소문자, 숫자로 구성된 문자열 (길이: 1 <= len <= 15)
    columnList = list(input())
    
    # 길이 15까지 *을 빈 자리에 채움
    for _ in range(15-len(columnList)) :
        columnList.append("*")
        
    # 매트릭스에 columnList를 append함
    matrix.append(columnList)
    
# 이제 matrix는 15*5의 고정 길이 matrix가 됨.
# 가로 길이만큼 반복
for col in range(15) :
    # 세로 길이만큼 반복
    for row in range(5):
        # "*"이 아닌 문자를 출력함
        if (matrix[row][col] != "*") :
            print(matrix[row][col], end="")